child_compassion/wizards/global_child_search.py

```diff
@@ -123,13 +123,19 @@ class GlobalChildSearch(models.TransientModel):
                  'completion_date_after', 'completion_date_before', 'local_id'
                  )
     def _compute_advanced_critieria_used(self):
-        self.advanced_criteria_used = (self.state_selected and
-                                       self.state_selected != 'Available') or \
-            self.chronic_illness or self.holding_gp_ids or \
+
+        advanced_criteria_used = self.chronic_illness or self.holding_gp_ids or \
             self.father_alive or self. mother_alive or \
             self.physical_disability or self.completion_date_after or \
             self.completion_date_before or self.local_id
 
+        if advanced_criteria_used:
```

Comment:
I don't see why doing the check this way is better than having all the conditions at the same place like how it was done before? I think it's more confusing now...

Reply:
It is necessary because before if we selected for example chronic_illness and mother_alive but the state was 'Available', this would make a simple search instead of an advanced search. I'll make it a bit more simple though